src/sentry/relay/config/__init__.py

```diff
@@ -305,7 +305,10 @@ def get_project_config(
     """
     with sentry_sdk.push_scope() as scope:
         scope.set_tag("project", project.id)
-        with metrics.timer("relay.config.get_project_config.duration"):
+        with (
+            sentry_sdk.start_transaction(op="get_project_config"),
```

Comment:
Existing observability on the actual build relies on the build tasks, which also has task-related data. The extra celery and Redis spans make inspecting `get_project_config` harder.